js/hitbtc3.js

```diff
@@ -1459,11 +1459,13 @@ module.exports = class hitbtc3 extends Exchange {
             }
             request['stop_price'] = this.priceToPrecision (symbol, stopPrice);
         }
-        const method = this.getSupportedMapping (market['type'], {
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('createOrder', undefined, params);
```

Comment:
We must use/pass the `market` variable here in the 2nd argument.

Reply:
I left out market because if I leave it in it thinks I'm trying to place a spot order and I'm unable to create a margin order since our naming convention for spot and margin orders is the same. Is there another way around this?

Reply:
@Dan-krm let's still specify the market arg in handleMarketTypeAndParams and let's do the following throughout hitbtc3 where market might be available:

```JavaScript
let [ marketType, query ] = this.handleMarketTypeAndParams ('createOrder', market, params);
```

The user will then have to add a `defaultType: margin` param-override to place an order with spot-margin market.

Reply:
> I'm trying to place a spot order and I'm unable to create a margin order since our naming convention for spot and margin orders is the same. Is there another way around this?

You can try this: 
- `createOrder (symbol, type, side, amount, price, { 'defaultType': 'margin' })`
- `node examples/js/cli hitbtc3 createOrder symbol type side amount price '{"defaultType":"margin"}' --verbose
`

Reply:
> The user will then have to add a `defaultType: margin` param-override to place an order with spot-margin market.

Okay sounds good I'll keep this in mind for the rest of the methods